lib/internal/validators.js

```diff
@@ -34,10 +34,18 @@ const {
 } = require('internal/util/types');
 const { signals } = internalBinding('constants').os;
 
+/**
+ * @param {number} value
```

Comment:
```suggestion
 * @param {unknown} value
```

Aren't these called on values that can be anything?

Reply:
True. Thanks for the review! I fixed it with a new commit